Firebase/Auth/Source/FIRAuth.m

```diff
@@ -438,7 +438,26 @@ - (instancetype)initWithAPIKey:(NSString *)APIKey appName:(NSString *)appName {
     _settings = [[FIRAuthSettings alloc] init];
     _firebaseAppName = [appName copy];
     #if TARGET_OS_IOS
-    UIApplication *application = [UIApplication sharedApplication];
+
+    static Class applicationClass = nil;
```

Comment:
Use existing function since FirebaseCore is always available to FirebaseAuth - https://github.com/firebase/firebase-ios-sdk/blob/03d26a76a0f7efb7ed3bb142a8e558fb68e4cccf/Firebase/Core/third_party/FIRAppEnvironmentUtil.m#L217

Reply:
Done, thanks.